eng/Version.Details.xml

```diff
@@ -35,6 +27,16 @@
       <Uri>https://github.com/dotnet/core-setup</Uri>
       <Sha>7d57652f33493fa022125b7f63aad0d70c52d810</Sha>
     </Dependency>
+    <!-- Change blob version in GenerateLayout.targets if this is unpinned to service targeting pack -->
+    <Dependency Name="Microsoft.WindowsDesktop.App.Ref" Version="3.1.0" Pinned="true">
```

Comment:
I thought these were no longer going to be produced unless we took explicit action to patch. Do we need to pin? cc @dagood

Reply:
Right, Core-SDK shouldn't have to pin as of https://github.com/dotnet/core-setup/pull/8827 (3.0), ported to 3.1 via https://github.com/dotnet/core-setup/pull/8881.

Reply:
I do see a 3.1.0 version of this package in the drop for 3.1 GA

Reply:
So we don't need to pin either core-setup pack? We currently depend on 3.1.1-servicing versions of both

Reply:
We need to depend on 3.1.0. It looks like the change didn't happen soon enough to not have to make these manual steps once again. But I think we don't need pinned=true if it's not being produced anymore?

Reply:
The "don't build targeting pack" infra only kicks in when building stabilized packages... this was *meant* to make sure we publish a new one for each `preview` release. I didn't realize we would go back to turning on prerelease versioning (with the `servicing` moniker) during the servicing lifecycle and didn't consider that scenario.

I guess we do, and we'll need to figure out a condition for pre-GA vs. servicing builds to patch the behavior. Ergh.

Reply:
Ah, that explains it, then. Thanks. I'll work on getting this green.

Reply:
Filed https://github.com/dotnet/core-setup/issues/8927 to track fixing.